master/src/main/java/org/apache/celeborn/service/deploy/master/clustermeta/ha/HARaftServer.java

```diff
@@ -492,11 +494,19 @@ public void updateServerRole() {
     }
   }
 
+  public Pair<String, String> addressToIpHostAddressPair(String address) {
```

Comment:
a little bit confusing, can u add java docs and test for this method?

Reply:
Move to Utils and add doc for this. Seems impossible mock object for Scala，so I do manual test for both Scenario.